emission/export/export.py

```diff
@@ -5,31 +5,38 @@
 import uuid
 import json
 import bson.json_util as bju
-
+import emission.storage.timeseries.timequery as estt
 import emission.storage.timeseries.abstract_timeseries as esta
-#import emission.storage.timeseries.cache_series as estcs
+import emission.storage.timeseries.cache_series as estcs
 
-def export(loc_entry_list, trip_entry_list, place_entry_list, ma_entry_list, user_id, file_name, ts):
-	first_place_extra_query = {'$and': [{'data.enter_ts': {'$exists': False}},{'data.exit_ts': {'$exists': True}}]}
-	first_place_entry_list = list(ts.find_entries(key_list=None, time_query=None, extra_query_list=[first_place_extra_query]))
-	logging.info("First place entry list = %s" % first_place_entry_list)
-	combined_list = ma_entry_list + loc_entry_list + trip_entry_list + place_entry_list + first_place_entry_list
+def export(ma_entry_list, user_id, ts, start_ts, end_ts):
```

Comment:
There are two ways to deal with the `ma_entry_list`.
- one is the current way, which is actually OK
- the other is to pass in a boolean `use_usercache` and read the `ma_entry_list` from the usercache if it is true

In either case, please move the new parameter to the end of the parameter list since by convention, we have the `user_id` be the first entry if it exists.

Reply:
New commit with the discussed changes located here. Please let me know if there is anything else visibly wrong within these files! 
https://github.com/e-mission/e-mission-server/pull/824/commits/4540a24875457962e345e028c8983e15a32db654